tensorflow/python/kernel_tests/unique_op_test.py

```diff
@@ -106,6 +107,49 @@ def testBoolV2(self):
     for i in range(len(x)):
       self.assertEqual(x[i], tf_y[tf_idx[i]])
 
+  @test_util.run_deprecated_v1
+  def testShapeInferenceV2(self):
+    """Test shape inference."""
+    x = np.random.randint(2, high=10, size=(3, 2, 1))
```

Comment:
Lets use an explicit array (or couple of arrays) here instead

Reply:
Updated to `np.arange(6).reshape(3, 2, 1)`